openapi.yaml

```diff
@@ -0,0 +1,58 @@
+openapi: "3.0.1"
+info:
+  title:
+    Ref: AWS::StackName
+  description: REST API for Follower Count
+paths:
+  /{twitter_handle}:
+    get:
+      responses:
+        200:
+          description: "200 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Empty"
+      x-amazon-apigateway-integration:
+        type: "aws"
+        credentials:
+          Fn::GetAtt: [RestAPIRole, Arn]
+        uri: "arn:aws:apigateway:us-east-1:dynamodb:action/Query"
+        responses:
+          default:
+            statusCode: "200"
+            responseTemplates:
+              application/json: |
+                #set($inputRoot = $input.path('$'))
+                {
+                "twitter_handle": $inputRoot.Items[0].twitter_handle.S,
+                "Items": [#foreach($item in $inputRoot.Items){
+                  "EpochTime": $item.EpochTime.N,
+                  "followers": $item.followers.N
+                }#if($foreach.hasNext),#end
+                #end
+                ]}
```

Comment:
As you're probably unaware, API gateway has a response payload limitation of 10MB. Given that this is invoked once an hour, that will lead to exceeding the service's limit in slightly less than 20 years. Please add pagination logic to avoid that happening; AWS services don't remove the need for us to consider bottlenecks in our code.

Reply:
You're right @QuinnyPig .  I have added a new commit to add pagination. 😄